internal/server/create_document.go

```diff
@@ -277,9 +313,43 @@ func translateInsertData(selectedTemplateInserts []sirius.Insert, documentTempla
 			}
 		}
 	}
+
+	sort.Slice(documentTemplateInserts, func(i, j int) bool {
+		return documentTemplateInserts[i].Handle < documentTemplateInserts[j].Handle
+	})
+
 	return documentTemplateInserts
 }
 
+func getSortedInsertKeys(selectedTemplateInserts []sirius.Insert) []string {
+	var documentInsertKeys []string
+	for _, in := range selectedTemplateInserts {
+		if !slices.Contains(documentInsertKeys, in.Key) {
+			documentInsertKeys = append(documentInsertKeys, in.Key)
+		}
+	}
+	// insert api response infrequently includes the key all
+	// note api keys are lowercase so this check is case-sensitive
+	if !slices.Contains(documentInsertKeys, "all") {
+		documentInsertKeys = append(documentInsertKeys, "all")
+	}
```

Comment:
I'll create a ticket to clean up the document template api response so it's more consistent with the inclusion of the all key and then this can be removed? Or conversely we could remove it from the api response and always handle it in the frontend